examples/app/log_layers_ecs.rs

```diff
@@ -77,7 +77,8 @@ impl tracing::field::Visit for CaptureLayerVisitor<'_> {
         }
     }
 }
-fn update_subscriber(app: &mut App, subscriber: BoxedSubscriber) -> BoxedSubscriber {
+/// Update Subscriber
```

Comment:
I don't really think this doc comment is helpful. Could it be improved or removed?

Reply:
I will remove it